submissions/SergeyShytikov/html-css-practice-task/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
```

Comment:
Use [prettier](https://prettier.io/) for auto-formatting code.

Reply:
Prettier was installed to my vscode as an extension and I think works well. Maybe you can explain what I need to correct?
<img width="238" alt="Screenshot 2021-09-14 at 23 13 12" src="https://user-images.githubusercontent.com/44546846/133327566-688e4dc3-943e-48e3-aa8f-46a687160f51.png">